src/components/ErrorBoundary/__snapshots__/ErrorBoundary.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Error Boundary renders without crashing and matches snapshot 1`] = `ShallowWrapper {}`;
```

Comment:
You can see here that this test doesn't actually work. As you're effectively testing nothing. Same as below.